R/experiment.R

```diff
@@ -278,7 +278,7 @@ Experiment <- R6::R6Class(
       dgp_params_list <- private$.combine_vary_params("dgp")
       method_list <- private$.get_obj_list("method")
       method_params_list <- private$.combine_vary_params("method")
-      fit_params <- purrr::cross2(dgp_params_list, method_params_list) %>%
+      fit_params <- cross(list(dgp_params_list, method_params_list)) %>%
```

Comment:
Thanks @jpdunc23! I looked into replacing `purrr::cross` with `tidyr::expand_grid()` very briefly before so can help if needed. Just let me know.

One subtle thing to note here is that if we simply replace `cross` with `tidyr::expand_grid()`, this will cause problems when naming the columns of the tibble if the method and DGP vary across parameters have the same name. We may want to append "dgp"/"method" to these names before calling `tidyr::expand_grid()` or find some other way to handle this case with care.